vehicle/nissan/types.go

```diff
@@ -77,15 +77,18 @@ type StatusResponse struct {
 }
 
 type Attributes struct {
-	ChargeStatus       float32   `json:"chargingStatus"`
-	RangeHvacOff       int       `json:"rangeHvacOff"`
-	BatteryLevel       int       `json:"batteryLevel"`
-	BatteryCapacity    int       `json:"batteryCapacity"`
-	BatteryTemperature int       `json:"batteryTemperature"`
-	PlugStatus         int       `json:"plugStatus"`
-	LastUpdateTime     Timestamp `json:"lastUpdateTime"`
-	ChargePower        int       `json:"chargePower"`
-	RemainingTime      *int      `json:"chargingRemainingTime"`
+	ChargeStatus          float32   `json:"chargeStatus"`
```

Comment:
Is this a general change or does it affect newer models only? If newer models only then this PR will break older models.

Reply:
We receive this kind of response for the cars connected via EVCC Nissan package, not Carwings. And I don't see any Nissan types anywhere in carwings.go. Do I miss something? Or misunderstand?

Reply:
This is what I'm seeing (not carwings)- there are more/different attributes:

```
{
	"id": "...",
	"chargePower": 3,
	"timeRequiredToFullFast": 0,
	"plugStatus": 1,
	"rangeHvacOn": 257,
	"batteryBarLevel": 240,
	"timeRequiredToFullNormal": 0,
	"batteryCapacity": 40000,
	"plugStatusDetail": 1,
	"timeRequiredToFullSlow": 0,
	"batteryLevel": 100,
	"rangeHvacOff": 265,
	"chargeStatus": 0,
	"lastUpdateTime": "2022-08-01T12:29:05Z"
}
```